xdsl/dialects/omp.py

```diff
@@ -141,6 +147,31 @@ class VariableCaptureKind(StrEnum):
     VLAType = "VLAType"
 
 
+class LoopWrapper(NoTerminator):
```

Comment:
Sorry, missed a bit of the discussion, is inheriting from traits compatible with how `has_trait` works?

Reply:
Yep